actionpack/test/dispatch/request_test.rb

```diff
@@ -597,9 +614,9 @@ class RequestParamsParsing < BaseRequestTest
   test "doesn't break when content type has charset" do
     request = stub_request(
       "REQUEST_METHOD" => "POST",
-      "CONTENT_LENGTH" => "flamenco=love".length,
+      "CONTENT_LENGTH" => "flamenco=love".length.to_s,
       "CONTENT_TYPE" => "application/x-www-form-urlencoded; charset=utf-8",
-      "rack.input" => StringIO.new("flamenco=love")
+      input: "flamenco=love"
```

Comment:
nit, let's use => for all the elements in the Hash, e.g `:input => "flamenco=love"`